src/Sylius/Bundle/ApiBundle/Creator/ImageCreatorInterface.php

```diff
@@ -18,5 +18,5 @@
 interface ImageCreatorInterface
 {
     /** @param array<mixed> $context */
-    public function create(string $ownerCode, ?\SplFileInfo $file, ?string $type, array $context = []): ImageInterface;
+    public function create(string $ownerIdentifier, ?\SplFileInfo $file, ?string $type, array $context = []): ImageInterface;
```

Comment:
```suggestion
    public function create(
        string $ownerIdentifier,
        ?\SplFileInfo $file,
        ?string $type = null,
        array $context = [],
    ): ImageInterface;
```
Since type is not required in the first place and it's a generic creator.